SENG-499-Company2-B01/Backend
# Add db init script for mongodb to create required collections

# Mongodb Init Script 
In this PR we created a simple initialization script which creates the initial schedule database, and all the required collections including: 
1. users 
2. classrooms 
3. courses
4. schedules 

## Files of interest (under mongodb directory) 
1. Dockerfile 
2. mongo-init.js 

## Test / Demo instructions 

1. Run docker-compose: 
`docker-compose up -d` 

2. Verify the mongo container is up and in a healthy state: 
`docker ps -a` 

3. Exec into the container: 
`docker exec -it backend_mongo_db_1 bash` then login to mongo `mongo -u <username> -p <password> ` 

4. Verify you can see schedule database: 
`show databases`, and then run `use schedule_db` 

5. Finally verify all collections exist: 
`show collections`
